packages/wdio-webdriver-mock-service/src/WebDriverMock.js

```diff
@@ -28,17 +28,45 @@ export default class WebDriverMock {
         this.command = new Proxy({}, { get: ::this.get })
     }
 
+    /**
+     * To allow random session IDs in url paths we have to set up a custom
+     * matcher that strips out the sessionID part from the expected url
+     * and actual url and replaces it with a constant session id
+     * @param   {String}   expectedPath path to match against
+     * @returns {Function}              to be called by Nock to match actual path
+     */
+    static pathMatcher (expectedPath) {
```

Comment:
I don't think the return arrow function part is needed here, also the string concatination is not needed. would the following work as well?
```
    static pathMatcher (expectedPath) {
            const sessionId = path.match(REGEXP_SESSION_ID)

            /**
             * no session ID found so we can check against expected path directly
             */
            if (!sessionId) {
                return path === expectedPath
            }

            /**
             * remove the session ID from expected and actual path
             * to only compare non arbitrary parts
             */
            expectedPath = expectedPath.replace(':sessionId', SESSION_ID)
            path = path.replace(sessionId[0].slice(1), SESSION_ID)
            return path === expectedPath
    }
```

Reply:
@erwinheitzman we need to return a function as we are using the function as follows:
```js
this.scope[method.toLowerCase()](WebDriverMock.pathMatcher(urlPath))
```
Nock expects a function as parameter.

Reply:
also I don't understand what you mean with

> also the string concatination is not needed

Reply:
I meant to use `path.replace(sessionId[0].slice(1), SESSION_ID)` instead of `path.replace(`${sessionId[0].slice(1)}`, SESSION_ID)`

Reply:
I think this will be the same and `sessionId[0].slice(1)` is not a concatenation

Reply:
Sorry, maybe I should have called it string templating. either way, the outcome will be the same so I am approving :)